common-service-core/src/test/java/org/zowe/apiml/util/CorsUtilsTest.java

```diff
@@ -25,6 +26,8 @@ class CorsUtilsTest {
     Map<String, String> metadata = new HashMap<>();
     List<String> defaultCorsMethods = List.of("GET", "HEAD", "POST", "PATCH", "DELETE", "PUT", "OPTIONS");
 
+    List<String> allowedEndpoints = Arrays.asList("/*/*/gateway/**", "/gateway/*/*/**", "/gateway/version");
```

Comment:
```suggestion
    List<String> allowedEndpoints = Arrays.asList("/gateway/**");
```
I know it is just a test, but at least to be consistent.